tests/test_command_line.py

```diff
@@ -0,0 +1,112 @@
+"""
+test_command_line.py: Test the command line interfaces of this program.
```

Comment:
Very trivial recommendation. You can use `CliRunner` class from within `click.testing` to test click CLI applications too. See example [here](https://github.com/Sage-Bionetworks/schematic/blob/develop/tests/test_cli.py). We can totally save this for another PR.

Reply:
Yeah, feel free to open an issue for this! My only concern is to make sure that CliRunner actually invokes the script from the command line, and doesn't just run the main method from within Python (just in case something causes the script execution to not work or something).